src/components/common/popup.jsx

```diff
@@ -13,57 +67,59 @@ const customStyles = {
     justifyContent: 'center',
   },
   content: {
-    top: 'auto',
-    left: 'auto',
-    right: 'auto',
-    bottom: 'auto',
     border: 'none',
     borderRadius: 'none',
     maxHeight: '100vh',
     overflow: 'auto',
     padding: '0',
     position: 'static',
+    width: '90%',
+    maxWidth: '75rem',
+    backgroundColor: 'transparent',
   },
 }
 
-const ModalHeader = styled.header`
-  display: flex;
-  flex-direction: row-reverse;
-  padding: 1.2em;
-`
+Modal.setAppElement('#___gatsby')
 
-const ModalContent = styled.div`
-  padding: 0 4em 4em;
-`
+class Popup extends React.Component {
+  render() {
+    const { children, onRequestClose, ...rest } = this.props
+    return (
+      <Modal
+        {...rest}
+        onRequestClose={onRequestClose}
+        parentSelector={this.getDocumentBody}
+        style={customStyles}>
+        <PopupWindow>
+          <PopupWindowContainer>
+            <IconButton onClick={onRequestClose}>
+              <CloseIcon />
+            </IconButton>
+            {children}
+          </PopupWindowContainer>
+        </PopupWindow>
+      </Modal>
+    )
+  }
 
-const IconButton = styled.button`
-  background-color: transparent;
-  border: none;
-  cursor: pointer;
-  outline: none;
+  componentDidMount() {
+    document.addEventListener('keydown', this.closeOnEscape)
+  }
+  componentWillUnmount() {
+    document.removeEventListener('keydown', this.closeOnEscape)
+  }
 
-  &:hover #closeIcon {
-    stroke: #ababab;
+  closeOnEscape = event => {
+    if (event.keyCode === ESC_KEY_CODE) {
```

Comment:
doesn't it handle ESC itself?

it supposed to be

>Accessible modal dialog component for React

Reply:
Very strange, I am sure that I've checked it, and think how bad they implement accessibility, but now it works. Fixed